.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "arrowParens": "avoid",
```

Comment:
This will clash with AirBNB lint config.

Reply:
I think. Assuming this is referring to arrow func param parens.

Reply:
Good shout